shortcut_composer/actions.py

```diff
@@ -244,6 +244,12 @@ def create_actions() -> List[ComplexAction]: return [
         active_color=QColor(110, 160, 235),
     ),
 
+    templates.PieMenu(
+        name="Pick Color Sampler blend (%)",
+        controller=controllers.ColorSamplerBlendController(),
+        values=[100, 95, 90, 80, 70, 60, 50, 40, 30, 20, 10, 5],
```

Comment:
Teraz ogarnąłem, że dla wartości PieMenu hardcodowanych w pliku akcji dalej da się je przestawiać w trybie edycji, ale będzie się to resetować na starcie krity, bo nie jest to nigdzie zapisywane. Będę musiał chyba wyłączyć tryb edycji gdy wartości są tak podane.

To jest bardziej komentarz dla mnie - Twój kod jest ok :)